extra-files/generate_tool_list_from_ga_workflow_files.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+## Christophe Antoniewski <drosofff@gmail.com>
+## https://github.com/ARTbio/ansible-artimed/tree/master/extra-files/generate_tool_list_from_ga_workflow_files.py
+## Usage example:
+## generate_tool_list_from_ga_workflow_files.py -w workflow1 workflow2 -o mytool_list.yml -l my_panel_label
```

Comment:
You can put the example (and author/url) in the argparse epilog:
https://docs.python.org/3/library/argparse.html#epilog
That way it's also visible to the user.